res_partner_attributes_add_SIN/__openerp__.py

```diff
@@ -1,28 +1,10 @@
 # -*- coding: utf-8 -*-
-##############################################################################
-#
-#    Odoo, Open Source Management Solution
-#    Copyright (C) 2010 - 2014 Savoir-faire Linux
-#    (<http://www.savoirfairelinux.com>).
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as
-#    published by the Free Software Foundation, either version 3 of the
-#    License, or (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# © 2016 Savoir-faire Linux
```

Comment:
@JulieSFL @dufresnedavid @jbchouinard AFAIK

(C) 2014 - The work was published the first time in 2014 (The copyright claim is from 2014 onward)
(C) 2014-2016 - The work was first published in 2014 and last updated in 2016 (The copyright claim is from 2014 onward)
(C) 2014,2015,2016 - The work was first published in 2014 and updated in 2015 and 2016 (The copyright claim is from 2014 onward)
(C) 2010-2014,2016 Not sure but the copyright claim is from 2010 onward.

The most simple way is to just put the year of the first publication.